src/member.rs

```diff
@@ -673,7 +697,27 @@ where
         let (consensus_exit, exit_stream) = oneshot::channel();
         let config = self.config.clone();
         let sh = self.spawn_handle.clone();
+        let (messages_for_network, mut messages_from_recovery) =
+            mpsc::unbounded::<(UnitMessage<H, D, MK::Signature>, Recipient)>();
+        let (responses_newest_tx, responses_newest_rx) = mpsc::unbounded();
+        self.tx_recovery = Some(responses_newest_tx);
         info!(target: "AlephBFT-member", "{:?} Spawning party for a session.", self.index());
+        let n_members = self.n_members;
+        let our_index = self.keybox.index();
+        let recovered_unit = recovery::recover_newest_unit::<H, D, MK::Signature>(
```

Comment:
I think the delay in the current version would even be `CATCH_UP_SECS + DelayConfig::unit_creation_delay(0)`, because Creator starts after the recovery phase. If we make all the components start right away (not after the recovery phase), then indeed the  `CATCH_UP_SECS` can play the role of `DelayConfig::unit_creation_delay(0)`, so I believe the simplest thing would be to just make `DelayConfig::unit_creation_delay(0)` equal to `0` in the default config. Otherwise you will need to make `0` a special case in `Creator` which is probably not that clean.

Reply:
Curretly we collect responses for the "newest unit created by us" request for `unit_broadcast_delay(0)` (which is `4s` by default)`, and then the creator postpones the creation of the first unit for `unit_creation_delay(0)` (which is `500ms` by default), so the first unit is created `4500ms` after starting the node.